css/styles.css

```diff
@@ -400,3 +409,129 @@ button,
 .d-grid {
   display: grid;
 }
+
+@media screen and (min-width: 992px) {
+  /* Body limited to 1100px */
+
+  html {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
+
+  body {
+    max-width: 1100px;
+  }
+
+  .body {
+    min-width: 480px;
+  }
+
+  /* Header */
+
+  .navbar {
+    display: none;
+  }
+
+  .navbar-desktop {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .navbar-desktop ul {
+    gap: 40px;
+  }
+
+  .headline {
+    background-image: url(../img/background/headerbg.svg);
+    margin-bottom: 80px;
+    background-repeat: no-repeat;
+  }
+
+  .headline-content {
+    max-width: 700px;
+    margin: 180px 120px 150px 120px;
+    gap: 20px;
+  }
+
+  .headline-content h2 {
+    margin-top: 10px;
+  }
+
+  /* Portfolio - Cards */
+
+  .work-card {
+    display: grid;
+    grid-template-columns: auto auto;
+    grid-template-rows: auto;
+    gap: 50px;
+  }
```

Comment:
- [ ] I think it would be better if you make the card image a little big bigger. As you can see in this image there are some differences between your and the original template. Please stick to the given template as much as possible.

| Original      | Yours |
  | ----------- | ----------- |
| ![Screen Shot 2021-12-01 at 3 15 07 PM](https://user-images.githubusercontent.com/69011963/144217399-d34e3d13-f683-4ee0-b602-d579c79286a2.png)|![Screen Shot 2021-12-01 at 3 14 00 PM](https://user-images.githubusercontent.com/69011963/144217447-578462a3-77a0-4d3f-bde5-45618e47c4c8.png)|

Reply:
@Rustamxon7 Thank you! Will do it ASAP. 😃